test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml

```diff
@@ -4,9 +4,10 @@ prepare_write_cmd: "cassandra-stress write cl=ALL n=50050075  -schema 'replicati
 stress_cmd: ["cassandra-stress write cl=QUORUM duration=11h  -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=250  -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=400200300..600200300 -log interval=15"]
 stress_read_cmd: ["cassandra-stress read cl=ONE duration=11h -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=250  -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..50050075 -log interval=5"]
 run_fullscan: ['{"mode": "table_and_aggregate", "ks_cf": "keyspace1.standard1", "interval": 5}']
-n_db_nodes: 4
```

Comment:
looks like 4 here for a reason. this test ```asymmetric cluster```
cc @juliayakovlev @fruch 


@pehala  does this change means that we are not supporting asymmetric with tablets

Reply:
the reason is same as all case we have 4 nodes, for make sure we can carry test is we lost one node (cause of SCT issue)

I don't see any other reason this specific case needed 4,

also the word asymmetric is a bit misleading, we have cause of asymmetric shards in each node, and this is not one of them

Reply:
and yes seem like this one is use only on the asymmetric shards cases.
but they are all random per node, 3 or 4, it's not much of a difference

Reply:
I think that by increasing it to 6, I would change the instance type to i4i.2xlarge.
Many/most of our customers use small instances because of this limitations and we should test with smaller instances as well.

Reply:
Changed the instances as well